images/gh-gl-sync/requirements.txt

```diff
@@ -1,24 +1,33 @@
-attrs==19.3.0
-boto3==1.16.18
-cffi==1.14.0
-coverage==5.2
-flake8==3.8.3
-gitdb==4.0.5
-importlib-metadata==1.6.1
+attrs==21.2.0
+boto3==1.19.0
+botocore==1.22.0
+certifi==2021.10.8
+cffi==1.15.0
+charset-normalizer==2.0.7
+coverage==6.0.2
+Deprecated==1.2.13
+flake8==4.0.1
+idna==3.3
+iniconfig==1.1.1
+jmespath==0.10.0
 mccabe==0.6.1
-more-itertools==8.4.0
-packaging==20.4
-pluggy==0.13.1
+packaging==21.0
+pluggy==1.0.0
 py==1.10.0
-pycodestyle==2.6.0
+pycodestyle==2.8.0
 pycparser==2.20
-pyflakes==2.2.0
-PyGithub==1.54
+pyflakes==2.4.0
+PyGithub==1.55
+PyJWT==2.3.0
+PyNaCl==1.4.0
 pyparsing==2.4.7
-pytest==5.4.3
-pytest-cov==2.10.0
-python-dateutil==2.8.1
-six==1.15.0
-smmap==3.0.4
-wcwidth==0.2.4
-zipp==3.1.0
+pytest==6.2.5
+pytest-cov==3.0.0
+python-dateutil==2.8.2
+requests==2.26.0
+s3transfer==0.5.0
+six==1.16.0
+toml==0.10.2
+tomli==1.2.1
+urllib3==1.26.7
+wrapt==1.13.2
```

Comment:
Also just curious how you came up with this list, which I assume is pinning all the versions for the same reason as using a hashed action.

Reply:
Since you ran it with those new deps, I still approve 😁